TWLight/emails/tasks.py

```diff
@@ -61,10 +61,6 @@ class CommentNotificationEmailOthers(template_mail.TemplateMail):
     name = "comment_notification_others"
 
 
-class ApprovalNotification(template_mail.TemplateMail):
```

Comment:
This email is still used in cases where the publisher isn't set to the access codes authorization method - we shouldn't be deleting it outright, just handling the access codes case better.